migration-guides/v6.0.0.md

```diff
@@ -269,4 +271,25 @@ Replace `hideCloseButton` with `preventClose`. If you still need the modal to cl
 <Modal preventClose shouldCloseOnEsc={true}>
   Modal content
 </Modal>
+```
+
+## 13. `<Table />` and `<SortableTable />` components now render a `<caption>` element with a `caption` prop it receives
+
+`caption` is a node prop and defaults to null. <caption> html tag is only rendered if `caption` is present.
+```jsx
+// Before
+<caption>My Awesome Table</caption>
+<Table>
+  Table content
+</Table>
```

Comment:
I don't think this is an appropriate Before example. The caption tag must be rendered **within** the table tag
![Image 2022-06-14 at 9 46 37 AM](https://user-images.githubusercontent.com/20426095/173592821-6bcc3496-be1d-49e7-a23c-07b416d2b7da.jpg)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption